app/controllers/admin/module_controller.rb

```diff
@@ -1,25 +1,13 @@
 # frozen_string_literal: true
 
 module Admin
-  class ModuleController < ApplicationController
-    layout 'admin'
-
-    alias current_user current_account # CanCanCan expects current_user.
-
-    rescue_from CanCan::AccessDenied, with: :redirect_to_index
-
+  class ModuleController < BaseController
```

Comment:
@amir-budaychiev what is the purpose of the `ModuleController`?

Reply:
@mitjok it's a good question, but I did a couple changes here, I didn't create that controller, you have to better ask the one who created @yivo 

but, as I understand - this is created for the inheritance of common characteristics admin's controllers.